### DISCUSSION TITLE
Check for (non)existence of HTML comments

### DISCUSSION THREAD

**Discussion by davidsneighbour** [cypress-io/cypress#16906] [Questions and Help]
I am new to Cypress and am building up my test cases. I had the idea of testing my static website generators use of templates (does a list page use the list template, does a single page with layout set loads the layout, etc) and planned to do that via comments in the HTML. But it seems it's not possible to test comments?

Does it make sense to use the XPath plugin for that or am I maybe using the wrong approach completely?

**Comment by tit** [comment]
You can not read html comments, because comments do not exist in DOM.

**Reply by davidsneighbour** [reply]
So using the XPath-plugin to get it might be the only option then.
